WordGame/ViewController.swift

```diff
@@ -23,14 +33,27 @@ class ViewController: UITableViewController {
         if allWords.isEmpty {
             allWords = ["fileError"]
         }
-        startGame()
+        loadLastGame()
     }
     
-   @objc private func startGame() {
+    @objc private func startGame() {
+        saveData()
         title = allWords.randomElement()
```

Comment:
Small note to say that, in real life, we shouldn't use the view controller title for this purpose (this is bad MVC). Instead, we'd have an object (a var in this case) that would store that random word and would also be the source for the navigation title update.